uuid.go

```diff
@@ -194,6 +194,14 @@ func (uuid UUID) String() string {
 	return string(buf[:])
 }
 
+// Bytes returns the bytes form of uuid, xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+// , or "" if uuid is invalid.
```

Comment:
This function cannot fail so there should just be a period at the end of line 197.

Reply:
done， It's ok ?  😁